src/plugins/intel_cpu/src/nodes/roi_pooling.cpp

```diff
@@ -453,6 +453,10 @@ void ROIPooling::initSupportedPrimitiveDescriptors() {
         impl_type = impl_desc_type::ref;
     }
 
+    if (impl_type != impl_desc_type::ref && refParams.src_prc == Precision::FP16) {
```

Comment:
let's have src_prc assignment in one place of the function

Reply:
@EgorDuplensky corrected